CriteoPublisherSdk/Sources/Context/CR_InternalContextProvider.m

```diff
@@ -19,14 +19,26 @@
 
 #import "CR_InternalContextProvider.h"
 
+#import <sys/utsname.h>
+
 @implementation CR_InternalContextProvider
 
+
 - (nullable NSString *)fetchDeviceMake {
-  return nil;  // TODO EE-1315
+  return @"Apple";
```

Comment:
![image](https://user-images.githubusercontent.com/3816099/100644212-52ca2a80-333b-11eb-98ad-bb25b47536fa.png)

Reply:
I think this needs a test ;p